src/Microsoft.ML.Featurizers/TimeSeriesImputer.cs

```diff
@@ -135,6 +151,12 @@ public enum ImputationStrategy : byte
             // Interpolate = 4, interpolate not currently supported in the native code.
         };
 
+        /// <summary>
+        /// What the filter strategy used is.
```

Comment:
Method by which columns are selected for imputing values

Reply:
Changed to your wording.